### DISCUSSION TITLE
Reference image can not be created. 15.2.4

### DISCUSSION THREAD

**Discussion by svenschedlbauer** [jomjol/AI-on-the-edge-device#2418] [Q&A]
Hi all,
I have installed version 15.2.4 new and now by camera is not working while I have to make the setup reference image.
Maybe somebody already had the same problem and now the solution or what I did wrong.

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/110197932/e8431a81-34c3-45c7-9fe3-f9ce2b6eb950)

Thanks in advance.

**Comment by caco3** [comment]
Please provide the log. You can access it undee `http://<IP>/log`
